playbooks/adhoc/uninstall.yml

```diff
@@ -39,7 +39,7 @@
         - atomic-openshift-master-api
         - atomic-openshift-master-controllers
         - atomic-openshift-node
-        - etcd
+        - "{% if is_atomic %}etcd_container{% else %}etcd{% endif %}"
```

Comment:
We disable the atomic host level etcd service during installation when is_containerized to avoid port conflicts so it shouldn't be used for anything already. I'd say disabling it during uninstall is fine.